src/sample/View.java

```diff
@@ -1,28 +1,27 @@
 package sample;
 
-import javafx.beans.value.ChangeListener;
-import javafx.beans.value.ObservableValue;
 import javafx.scene.Node;
 import javafx.scene.Scene;
 import javafx.scene.control.Button;
 import javafx.scene.control.CheckBox;
 import javafx.scene.control.TextField;
 import javafx.scene.layout.GridPane;
 import javafx.scene.text.Text;
-import javafx.scene.text.TextAlignment;
 import javafx.stage.Stage;
 import sample.models.Point;
 import java.util.function.Consumer;
 
 public class View
 {
+    static Stage mainGame;
     private static View instance = null;
     GridPane gridPane;
     int gridSize = Constants.DEFAULT_GRIDSIZE;
 
     // Constructor must be private for the Singleton pattern.
     View()
```

Comment:
Why'd we get rid of our singleton here

Reply:
This may be something we need to discuss because I am not sure.

Reply:
I still have this added line in my code so I am not sure on this one what to resolve